### PR TITLE
docs: add example of testing React Navigation

### PR DESCRIPTION
**Pull request by cross19xx**
### Summary

This PR includes
- setup an example of testing React Navigation navigator with default screen
- integration with the website

Part of #280

### CODE REVIEW

File: website/i18n/en.json [hunk 1]
```diff
@@ -21,6 +24,7 @@
     },
     "categories": {
       "Introduction": "Introduction",
+      "Examples": "Examples",
```
Comment (thymikee) [inline website/i18n/en.json:27]: let's put Examples below API Reference

File: docs/ReduxIntegration.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+id: redux-integration
+title: Redux Integration
+---
+
+Codesandbox provided as proof.
+
+<iframe
+  src="https://codesandbox.io/embed/new?codemirror=1"
```
Comment (thymikee) [inline docs/ReduxIntegration.md:9]: I requested creating an account, will update this PR once done

Comment (thymikee) [reply docs/ReduxIntegration.md:9]: We have an account now. So a step forward to this, would be for you to create a codesandbox with Redux example, which I could then fork to the official account

File: website/i18n/en.json (hunk 1 above)
Comment (cross19xx) [reply website/i18n/en.json]: Done

**Comment by thymikee** [comment]
We'll also need to set up running tests for examples, let's do it in a followup

**Pull request merged**